apps/omg_eth/test/fixtures.exs

```diff
@@ -24,13 +24,23 @@ defmodule OMG.Eth.Fixtures do
   deffixture eth_node do
     {:ok, exit_fn} = Eth.DevNode.start()
```

Comment:
that should go away?

Reply:
Actually can't remove this _just_ yet. Other tests are using these fixtures.